src/ui/components/NetworkConnectionError.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {Linking, Text, View, StyleSheet, Image, Platform} from 'react-native';
+import networkError from 'image/network/networkError.png';
+import {Button, Modal} from '@ui/library';
+import {standardPadding} from '@ui/styles';
+import {useNetInfo} from '@react-native-community/netinfo';
```

Comment:
Let's wrap this into a custom hook, later we can add more features like refetching queries when reconnecting to the internet.